feature1/A.java.txt

```diff
@@ -1,2 +1,3 @@
 feature1
 edited feature1
+code related to feature 2
```

Comment:
ok, you decided your line to be after my line
kindly confirm
if all ok then kindly proceed to merge, your PR looks good to me